pkg/util/azureclient/mgmt/redhatopenshiftpreview/openshiftclusters.go

```diff
@@ -0,0 +1,53 @@
+package redhatopenshiftpreview
```

Comment:
shall we just admit defeat and go to pkg/util/azureclient/mgmt/redhatopenshift/2021-01-31-preview/redhatopenshift ?